tests/testthat/test-generate_example_forecast_data.R

```diff
@@ -0,0 +1,69 @@
+library(dplyr)
+
+# These tests are designed to help devs check their work when updating the logic that generates hubExample's
+# data files. Because these scripts (i.e., the contents of data-raw/) are not included in the hubExample
+# package itself, the tests will not run during R checks.
+skip_if(is_checking())
+
+# Source the script that generates the example forecast data so we can test the functions
+source(file.path(forecast_data_path, "generate_example_forecast_data.R"))
+
+
+test_that("forecast_outputs dataset is generated correctly", {
```

Comment:
i can sense your burning anguish at the lack of parameterized tests here.

Reply:
It's true--testing in general is not the best in R imo (though snaps are pretty great!)